О павергейме.

dred1792 opened the pull request:
В повергейм добавляется понятие о ролькоханте (и оно же выведено из под офицеров СБ дабы не было дублирования). В пункт правил персонажа добавлены подпункты о страхе смерти/боли/последствий своих действий и о запрете к маниакальной тяге к убийствам/суициду.

dred1792 committed:
Merge remote-tracking branch 'refs/remotes/TauCetiStation/master'

dred1792 committed:
Merge pull request #1 from TauCetiStation/draft

Draft

dred1792 committed:
О ПГ и схожем.

dred1792 commented:
В данном ПРе присутствуют правки предложенные Сквазабром.

skvazarb22 commented on index.md:
```diff
@@ -84,7 +85,10 @@ _Не будь мудаком. Это основное правило повед
 1. Если игра не выдала вам роль антагониста — вы не являетесь им. Это, впрочем, не запрещает вам быть коррумпированным главой/офицером, красть, заниматься мелкими грабежами и тому подобным. Однако, следует придерживаться некоторых рамок. В общем случае правило следующее — целью ваших действий не должно быть надолго вывести другого игрока из игры. Вашей целью не должны быть предметы высокого риска ...
 
-1. Отыгрыш персонажа, принадлежащего к какой-либо инопланетной расе, должен учитывать особенности ее лора. 
+1. Отыгрывайте реалистичных и естественных персонажей в соответствии с лором игры (в том числе в соответствии с профессией, расой, режимом игры).
```
Я только сейчас заметил, что в качестве хедера раздела присутствует почти идентичная формулировка 
> Старайтесь, чтобы персонаж, за которого вы играете, был хоть в какой-то мере правдоподобен

"реалистичность" - близкий синоним "правдоподобности", так что это по сути одно и то же, но добавляемая формулировка явно лучше, так что надо удалить этот заголовок и перенести на его место этот пункт, чтобы не было дублирование одно и того же смысла в одном разделе.

dred1792 replied:
Ну тут если честно в одного решение принимать не буду. Ибо не считаю подобное дублирование чем-то критичным. Как вариант, можно из пункта просто убрать слово "реалистичных", оставив по итогу формулировку 

> "Отыгрывайте естественных персонажей в соответствии с лором игры (в том числе в соответствии с профессией, расой, режимом игры)"

skvazarb22 replied:
>не считаю подобное дублирование чем-то критичным

Всмысле, у нас через пол экрана в правилах повторяются формулировки, которые совпадают на 95%, как это можно это считать не "критичным".

>можно из пункта просто убрать слово "реалистичных", оставив по итогу формулировку

Можно подумать, что слово "естественных" куда-то далеко ушло по смыслу. Мы тут без лингвиста не разберемся чем в контексте ссочки отличается "правдоподобный", "реалистичный" и "естественный" персонаж. Если мы хотим сохранить заголовок в качестве общего предисловия, тогда уж просто

> Отыгрывайте персонажей в соответствии с лором игры (в том числе в соответствии с профессией, расой, режимом игры)

И, разумеется, этот пункт в этом случае просто обязан идти сразу за этим заголовком, так как они между собой напрямую логически связаны.

DantezZz replied:
>Старайтесь, чтобы персонаж, за которого вы играете, был хоть в какой-то мере правдоподобен.

Эта сноска с мелким шрифтом служила неким костылём, на который как раз таки и ссылалась администрация при варнах за самоубийства и прочие игнорирование боли, ибо это **нереалистично**.  Теперь же, с внесением данных правок, смысла в этом примечании становится гораздо меньше.

dred1792 committed:
Правки по фидбеку.

Убирается являющиеся ранее костылём "Старайтесь, чтобы персонаж, за которого вы играете, был хоть в какой-то мере правдоподобен.". Пункт который был 8м, ныне 1й.

dred1792 commented:
Внёс правки по фидбеку. Касательно того про что Скваз изначально писал допёр только вчера.

volas commented:
Для референса, основное обсуждение было в https://github.com/TauCetiStation/Rules/pull/28
